src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -343,6 +350,11 @@ public class Feature
         /// </summary>
         public static Feature ServerExtractsUsernameFromX509Certificate => __serverExtractsUsernameFromX509Certificate;
 
+        /// <summary>
+        /// Gets the server returns retryable writeError label feature.
+        /// </summary>
+        public static Feature ServerReturnsRetryableWriteErrorLabelFeature => __serverReturnsRetryableWriteErrorLabelFeature;
```

Comment:
Don't use "Feature" in the property name:

    public static Feature ServerReturnsRetryableWriteErrorLabel => __serverReturnsRetryableWriteErrorLabel;

Reply:
done